### ISSUE TITLE
Definition/use of GO:0040035 hermaphrodite genitalia development

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/go-ontology#23531]
### Discussed in https://github.com/geneontology/helpdesk/discussions/399

<div type='discussions-op-text'>

<sup>Originally posted by **suzialeksander** June 15, 2022</sup>
From Helpdesk email: 

> Dear curators,
> 
> Thank you for providing this excellent resource which I have been using for almost 15 years now. After this long period, I finally encountered a problem with Gene Ontology. In a recent manuscript, we reported some results of a Gene Ontology analysis for C. elegans genes. One of the terms that came up was 'hermaphrodite genitalia development' (GO:0040035).
> 
> One of the reviewers remarked:
> 
> "NB (page 7): Nematodes don’t have hermaphrodite genitalia. Hermaphroditic nematodes have female genitalia. 
> This looks like someone else’s terminology, but I think the wording could still be upgraded here."
> 
> I asked several colleagues and they all agreed with the reviewer's view.
> 
> Therefore, I would like to bring your attention to this issue and I would suggest to assess, whether a change in the Gene Ontology would be possible and appropriate. If you need another expert opinion, I would recommend to ask Paul Sternberg who is also listed as PI on the GO website.</div>

**Comment by suzialeksander** [comment]
Note @cmungall commented on the discussion: 
>it's in wbbt (which is linked indirectly via go->uberon->wbbt)
>https://www.ebi.ac.uk/ols/ontologies/wbbt/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FWBbt_0005178&lang=en&viewMode=PreferredRoots&siblings=false

**Comment by cmungall** [comment]
> But the question remains whether hermaphrodite genitalia development is a gonad development

good catch

We can see the general classification of these terms here:

<img width="865" alt="image" src="https://user-images.githubusercontent.com/50745/175186375-f252319f-58b3-4c6d-88ee-977db5860de9.png">

There are a number of oddities here, making a generalized system across all metazoa is hard

**Comment by raymond91125** [comment]
Medicine Net: Genitalia: The male or female reproductive organs. The genitalia include internal and external structures. The female internal genitalia are the ovaries, Fallopian tubes, uterus, cervix, and vagina. The female external genitalia are the labia minora and majora (the vulva) and the clitoris. The male internal genitalia are the testes, epididymis, and vas deferens. The male external genitalia are the penis and scrotum.
Thus gonad development should be part_of genitalia development.

**Comment by vanaukenk** [comment]
I've looked at the annotations from WB.
In many cases, it looks like we've been co-annotating to 'hermaphrodite genitalia development' (or morphogenesis) and a more specific term like 'vulval development' (or morphogenesis).
So, if we obsoleted 'hermaphrodite genitalia development' (and morphogenesis) and made sure that 'vulval development' (and morphogenesis) are children of 'female genitalia development', I think we'd be okay.  
Whether or not we should broaden the definition of 'vulval development' to try to be more inclusive of other metazoans or update the name of this term to be more specific, e.g. 'nematode vulval development'  is TBD.
Does that get us closer to a workable solution?
